code/modules/mob/living/carbon/human/species_types/sap.dm

```diff
@@ -0,0 +1,214 @@
+//Posh plant-based humanoids with woody skin. Their mood and by extension nutrition is determined by how fancy they are.
+/datum/species/sap
+	name = "Sap"
+	id = "sap"
+	limbs_id = "plant"
+	default_color = "59CE00"
+	species_traits = list(MUTCOLORS, EYECOLOR, NO_UNDERWEAR)
+	mutant_bodyparts = list("canopy")
+	default_features = list("mcolor" = "59CE00", "canopy" = "Oakley Traditional")
+	attack_verb = "smashed"
+	attack_sound = "genhit"
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	say_mod = "demeans"
+	speedmod = 1 //Though strong, saps are posh and insist on walking everywhere to maintain class
+	brutemod = 0.9 //Woody skin means that blunt attacks and the like are less effective
+	burnmod = 1.1
+	heatmod = 1.5
+	siemens_coeff = 0.4 //They're plants and are less vulnerable to shocks
+	exotic_blood = "ez_nut"
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/plant
+	disliked_food = MEAT | DAIRY
+	liked_food = VEGETABLES | FRUIT | GRAIN
+	var/fanciness = 0 //Percentage of fanciness, determined by clothing worn. Below 50% fanciness you begin to starve, and above it you are nourished!
+	var/list/fancy_clothing_info //Used so saps can track their fanciness in an alert
+
+
+	var/static/list/despicable_clothing_typecache = typecacheof(list(\
```

Comment:
All these lists are quite awful, not worth having 5 typechaches for your snowflake "fancy"(what the fuck is even this or why does it exist) thing

Reply:
It exists to give the race an interesting gimmick like plasmamen have; typecaches are only one way of doing it and I could also add a variable to `/obj/item/clothing` that serves the same purpose if it's that bad